zeebe/topology/src/main/java/io/camunda/zeebe/topology/api/ScaleRequestTransformer.java

```diff
@@ -20,10 +22,16 @@
 public class ScaleRequestTransformer implements TopologyChangeRequest {
 
   private final Set<MemberId> members;
+  private final int newReplicationFactor;
   private final ArrayList<TopologyChangeOperation> generatedOperations = new ArrayList<>();
 
   public ScaleRequestTransformer(final Set<MemberId> members) {
+    this(members, USE_CURRENT_REPLICATION_FACTOR);
+  }
+
+  public ScaleRequestTransformer(final Set<MemberId> members, final int newReplicationFactor) {
```

Comment:
:thought_balloon: Why not keep the `Optional<Integer>` instead of introducing the `USE_CURRENT_REPLICATION_FACTOR` constant?